src/commands/ExecCommand.js

```diff
@@ -38,6 +38,7 @@ export default class ExecCommand extends Command {
   getOpts(pkg) {
     return {
       cwd: pkg.location,
+      shell: true,
```

Comment:
I'm not sure this property is necessary, as [execa](https://github.com/sindresorhus/execa) is handling this for us.